### PR TITLE
Fix rpm

### PR DESCRIPTION
**Pull request by aashish24**
- [x] Mostly bringing changes from release to master
- [x] Updated to odd number for master

@sankhesh @chaosphere2112 please review

### CODE REVIEW

**Comment by aashish24** [comment]
@williams13 I need to get this merged before I would like to create a release branch

File: CMakeLists.txt [hunk 1]
```diff
@@ -821,30 +821,31 @@ add_custom_target(wrappers ALL DEPENDS
...
 set(CPACK_PACKAGE_NAME "UVCDAT")
-set(CPACK_PACKAGE_VERSION_MAJOR "1")
-set(CPACK_PACKAGE_VERSION_MINOR "4")
+set(CPACK_PACKAGE_VERSION_MAJOR "2")
+set(CPACK_PACKAGE_VERSION_MINOR "3")
 set(CPACK_PACKAGE_VERSION_PATCH "0")
 set(CPACK_PACKAGE_VERSION ${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH})
 set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/docs/README.txt")
 set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/docs/Legal.txt")
 set(CPACK_DEBIAN_PACKAGE_MAINTAINER "Aashish Chaudhary") #required
```
Comment (durack1) [inline CMakeLists.txt]: correct spelling of "PACAKGE" below - line 835

**Comment by aashish24** [comment]
thanks @durack1 fixed it.

**Comment by durack1** [comment]
@aashish24 on a related but separate note, how close are `*.dmg` UV-CDAT packages for OS X? Could these be implemented as a distribution option soon?

**Comment by williams13** [comment]
Okay! Please let me know when you have created the release branch.

**Comment by aashish24** [comment]
> @aashish24 on a related but separate note, how close are *.dmg UV-CDAT packages for OS X? Could these be implemented as a distribution option soon?

post 2.4, yes.

**Comment by sankhesh** [comment]
Looks good but the best validation is to generate the package and test.

**Comment by aashish24** [comment]
> Looks good but the best validation is to generate the package and test.

@sankhesh sounds reasonable @chaosphere2112 can you try to create an RPM using this branch?

**Comment by aashish24** [comment]
@sankhesh if I don't hear back from @chaosphere2112 I may merge it since I need it for the release branch. If we find some issues then we can push it to release or master or both.

**Comment by chaosphere2112** [comment]
@aashish24 I'll take a run at packaging an RPM shortly.

**Comment by chaosphere2112** [comment]
@aashish24 Ran into some build issues, taking a bit longer than anticipated. Wasn't running a new enough cmake, apparently.

**Comment by chaosphere2112** [comment]
... and our network is dieing horribly right now

**Comment by aashish24** [comment]
@chaosphere2112 any update?

**Comment by chaosphere2112** [comment]
@aashish24 Finally got my build working. I should hopefully be able to run cpack momentarily.

**Comment by aashish24** [comment]
Awesome! Thanks @chaosphere2112 for the update

**Comment by chaosphere2112** [comment]
Tried packaging, but wound up with these requirements:

```
uvcdat-1.4.0-1.x86_64 requires libQtWebKit.so.4()(64bit)
uvcdat-1.4.0-1.x86_64 requires libQtDeclarative.so.4()(64bit)
uvcdat-1.4.0-1.x86_64 requires /home/sam/work/builduvcdat/install/bin/python2.7
uvcdat-1.4.0-1.x86_64 requires /usr/local/bin/python
uvcdat-1.4.0-1.x86_64 requires /home/sam/work/builduvcdat/install/bin/python
```

**Comment by chaosphere2112** [comment]
Ugh, looks like I was on the wrong branch. Going to repackage.

**Comment by chaosphere2112** [comment]
Ran into some packaging errors, doing a clean build on the proper branch to see if that fixes it. Won't be done till after I'm out for the day (which is now), so we'll pick up on this tomorrow.

**Comment by aashish24** [comment]
thanks @chaosphere2112

**Comment by chaosphere2112** [comment]
Ran into the same packaging errors. [Here's][1] the log that was dumped.

[1]: https://gist.github.com/chaosphere2112/cd36a0ea423069c021ba

**Comment by chaosphere2112** [comment]
I found [this][1] bug report that mentions something similar to this issue; I'm running CMake 2.8.12, though, and they said it should have been in 2.8.8.

**Comment by aashish24** [comment]
thanks @chaosphere2112. Unless we are doing something, wrong here, I would suggest we merge this branch and follow up with CMake develoepers on this issue. Sounds reasonable?

**Comment by aashish24** [comment]
@chaosphere2112 I am not seeing the twice error/warning. My problem is with the freetype: 

failed to create symbolic link '/home/chaudhary/tools/uvcdat/build_debug/_CPack_Packages/Linux/RPM/UVCDAT-2.3.0-Linux//home/chaudhary/tools/uvcdat/build_debug/install/Externals/include/freetype': No such file or directory

**Comment by aashish24** [comment]
@chaosphere2112 both of these issues seems related but are not fault of this branch. I am merging this one and creating tag points.

**Pull request merged**